src/aviary/tools/utils.py

```diff
@@ -69,7 +69,6 @@ async def __call__(
                 "type": "function",
                 "function": {"name": tool_choice.info.name},
             }
-            expected_finish_reason = {"stop"}  # TODO: should this be .add("stop") too?
```

Comment:
I never understood why the `finish_reason` was `"stop"` when we'd directly specified to call a `Tool`.

Turns out it was a `litellm` bug fixed sometime between `litellm==1.71.1` and `litellm==1.72.0`